geometry_msgs/Pose targetPose
std_msgs/UInt16 velocity
---
bool success

